src/odemis/driver/test/actuator_test.py

```diff
@@ -68,27 +68,25 @@ def test_speed(self):
 
 
 class MultiplexOneTest(unittest.TestCase, simulated_test.ActuatorTest):
-
     actuator_type = MultiplexActuator
 
     def setUp(self):
         self.dependency = tmcm.TMCLController(name="test", role="test",
-                                         port="/dev/fake3",
-                                         axes=["a", "b"],
-                                         ustepsize=[5.9e-9, 5.8e-9],
-                                         rng=[[-1e-3, 1e-3], [0, 1e-3]],
-                                         refproc="Standard")
+                                              port="/dev/fake3",
+                                              axes=["a", "b"],
+                                              ustepsize=[5.9e-9, 5.8e-9],
+                                              rng=[[-1e-3, 1e-3], [0, 1e-3]],
+                                              refproc="Standard")
```

Comment:
Please, please, don't reformat the entire file whenever you have some change in it. It hides your "real" changes.
If the format is really terrible, do the reformatting as a separate first commit.
For this PR, let's not worry about it, and leave it like that, but avoid doing it in the future.

Reply:
Yes I thought indeed that went wrong at some point. It was never my intention to reformat the whole file only the part that I did the modification in. Somehow my IDE suggested i'd reformat the whole file and I clicked that too quickly I guess.
Thanks for reviewing and approving eventually.